### PR TITLE
Added "cjk" analyzer.

### LINKED ISSUE #592: [BUG] Missing support for "cjk" analyzer
### What is the bug?
For analyzer "cjk" exists no method.

Via REST/JSON I can use the analyzer.

### How can one reproduce the bug?
```java
            // create an index
            CreateIndexRequest createIndexRequest = new CreateIndexRequest.Builder().index(index)
                                                                                    .settings(b -> b
                                                                                        .analysis(ab -> ab
                                                                                            .analyzer("ko_analyzer", aab->aab
                                                                                       >>>>     .cjk()  <<<<
                                                                                            )
                                                                                        )
                                                                                    ).build();
            CreateIndexResponse createIndexResponse = client.indices().create(createIndexRequest);
```

### What is the expected behavior?
That "cjk" analyzer is supported like all the other analyzers.

### What is your host/environment?
AWS Opensearch Serverless / java-client 2.6.0

### Do you have any screenshots?
No.

### Do you have any additional context?
No.

### PR DESCRIPTION
**Pull request by MikePieperSer**
### Description
Add support for "cjk" analyzer.

### Issues Resolved
Closes #592 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ### Added
 - Document HTTP/2 support ([#330](https://github.com/opensearch-project/opensearch-java/pull/330))
+- Added support for "cjk" analyzer ([#604](https://github.com/opensearch-project/opensearch-java/pull/604))
```
Comment (reta) [inline CHANGELOG.md:8]: Could you please move it under `## [Unreleased 2.x] ` section? Thank you.

File: java-client/src/main/java/org/opensearch/client/opensearch/_types/analysis/CjkAnalyzer.java [hunk 2]
```diff
@@ -0,0 +1,201 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
```
Comment (reta) [inline java-client/src/main/java/org/opensearch/client/opensearch/_types/analysis/CjkAnalyzer.java:9]: Please remove Elasticsearch related headers, this is new file

**Review by reta — DISMISSED** [review]

**Comment by MikePieperSer** [comment]
Sorry, gave my complete name, again. Shall I fix it and force push?

**Comment by reta** [comment]
> Sorry, gave my complete name, again. Shall I fix it and force push?

Yes please, thank you

**Review by VachaShah — DISMISSED** [review]
This is great, thanks @MikePieperSer!

It would also be helpful if we have these code snippets listed in the user guide in an upcoming PR.

File: java-client/src/main/java/org/opensearch/client/opensearch/_types/analysis/Analyzer.java [hunk 3]
```diff
@@ -90,6 +90,8 @@ public enum Kind implements JsonEnum {
 
 		Whitespace("whitespace"),
 
+        Cjk("cjk"),
```
Comment (dblock) [inline java-client/src/main/java/org/opensearch/client/opensearch/_types/analysis/Analyzer.java:93]: Tabs vs. spaces!

**Comment by MikePieperSer** [comment]
Can we also backport this to 2.x please?

**Review by reta — APPROVED** [review]

**Review by VachaShah — APPROVED** [review]

**Pull request merged**